pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -374,6 +378,28 @@ def _get_image_stream_name(assembly_type: assembly.AssemblyTypes, arch: str):
         go_arch_suffix = go_suffix_for_arch(arch)
         return f'4-dev-preview{go_arch_suffix}' if assembly_type == assembly.AssemblyTypes.PREVIEW else f'release{go_arch_suffix}'
 
+    async def sync_rhcos_srpms(self, assembly_type, data):
+        # Sync potential pre-release source on which RHCOS depends. See ART-6419 for details.
+        major, minor = util.isolate_major_minor_in_group(self.group)
+        if (major > 4 or (major == 4 and minor >= 13)) and assembly_type in [assembly.AssemblyTypes.CANDIDATE, assembly.AssemblyTypes.PREVIEW]:
```

Comment:
Please remove the version logic. Type preview or Candidate is enough.

Reply:
updated